components/settings/Settings.tsx

```diff
@@ -1,8 +1,10 @@
+
 import React, { useEffect, useState } from 'react';
 // import { useQuery } from 'react-query';
 import useUpdateSettings, { useFetchSettings } from '../../services/settings';
 import Icon from '../common/Icon';
 import SelectField, { SelectionOption } from '../common/SelectField';
+import { ScraperTypes } from "../../types.d"
```

Comment:
Nextjs automatically imports types from the `types.d` file. No need to import the types.